### ISSUE TITLE
Better documentation

### ISSUE THREAD

**Issue by ghistes** [naggie/dstask#108]
Hi,

I just discovered this project and believe it has great potential.

However I only dabbled a little bit in taskwarrior and so (as maybe other newcomers) are not so familiar with the terms that you use.

For example I had no idea what a "context" was, I had to go to the taskwarrior site to learn about that as you seem to  cover your terminology at all.

I still have to clue what an "antitag" might be.

So as a new user I would wish for some example use-cases and a brief explanation of the concepts and the terminology.

**Comment by naggie** [comment]
Agreed -- thanks for mentioning what is not clear from the perspective of a new user -- a perspective I can't have :)

PR would be appreciated if you get time. If not no prob.

**Comment by ghistes** [comment]
I think a wiki (like e.g. https://github.com/jarun/nnn/wiki) would be a good way to collect use-cases.
